db/data/data.test.js

```diff
@@ -2,9 +2,9 @@ const instructorsData = require('./instructors.json');
 const offeredBysData = require('./offeredBys.json');
 const testimonialsData = require('./testimonials.json');
 
-const instructorsModel = require('../models/instructorsModel.js');
-const offeredBysModel = require('../models/offeredBysModel.js');
-const testimonialsModel = require('../models/testimonialsModel.js');
+const instructorsModel = require('../models.js').InstructorsModel;

```

Comment:
You could also do:
const models = require('../models.js');
const instructorsModel = models.InstructorsModel;
const offeredBysModel = models.OfferedBysModel; 
etc.